docs/docs/installation/github.md

```diff
@@ -28,13 +28,23 @@ jobs:
 ```
 
 
-if you want to pin your action to a specific release (v2.0 for example) for stability reasons, use:
+if you want to pin your action to a specific release (v0.23 for example) for stability reasons, use:
 ```yaml
 ...
     steps:
       - name: PR Agent action step
         id: pragent
-        uses: Codium-ai/pr-agent@v2.0
+        uses: docker://codiumai/pr-agent:0.23-github_action
+...
+```
+
+For enhanced security, you can also specify the Docker image by its digest:
```

Comment:
remove the "enhanced security".
its a bit confusing. It will not protect you from cyber attacks.
you mean to say that it ensures without a doubt that the docker will be freezed (although in practice the v0... models are freezed anyway)




```
you can also specify the Docker image by its digest:
```

Reply:
I mean specifying the Docker image digest enhances security because it is effective in preventing supply chain attacks.
If the term "enhanced security" is confusing, would it be clearer to phrase it as "to prevent supply chain attacks"?

Reply:
you know what, its ok. thanks for the PR